BMS_AttributeCheck.c

```diff
@@ -0,0 +1,80 @@
+/*
+ *
+ ***************************************************************************************************
+ * Author:             Harshitha Subramani
+ * Date:               23/02/2020  
+ * Filename:           telecolor_test.h
+ *
+ ****************************************************************************************************
+*
+***************************************************************************************
+* Includes
+***************************************************************************************
+*/
+
+#include <stdio.h>
+#include <assert.h>
+#include "BMS_AttributeCheck.h"
+
+/*
+***************************************************************************************
+* Methods
+***************************************************************************************
+*
+ *This Function Checks if the Battery attribute value read is less than the defined Minimum limit
+ * \param     attribute_value             Battery reading
+ * \param     attribute                   This structure states attribute name , minimum and maximum limit
+ * \return                                returns 1 if the Battery reading is less then the defined Minimum limit else returns 0     */         
+
+int Check_Attributelow(float attribute_value,struct attribute s_attribute){
+    
+    if(attribute_value < s_attribute.attribute_minvalue)
+    {
+        printf("%s is low\n",s_attribute.attribute_name);
```

Comment:
When this needs to take care of multiple display-languages, the printf can get complicated. Also duplicated in the next function. Can you alter the return value to include the low-high information, so the printf is separate?